Tests/TuistGeneratorTests/Settings/DefaultSettingsProviderTests.swift

```diff
@@ -499,7 +499,10 @@ private extension XCTestCase {
                            file: StaticString = #file,
                            line: UInt = #line) {
         for (key, expectedValue) in second {
```

Comment:
for this particular test `nil` is a valid value to compare against, the warning is generated due to the string interpolation.

Perhaps to mitigate this warning, we can either do `String(describing:)` or `let resultDescription = results.map { "\($0)" } ?? "nil"` and use that in the `"\(key):\(resultDescription) does not match....`

Reply:
thanks